nbconvert/preprocessors/csshtmlheader.py

```diff
@@ -28,8 +30,9 @@ class CSSHTMLHeaderPreprocessor(Preprocessor):
                               help="CSS highlight class identifier"
     ).tag(config=True)
 
-    style = Unicode('default',
-            help='Name of the pygments style to use'
+    style = Union([Unicode('default'), Type(klass=Style)],
```

Comment:
This change should be done for the latexPreprocessor too.

Reply:
Is this good, or is there an issue with this that should be attacked in this PR?